enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -230,6 +231,10 @@ public void obligation( long txId ) throws IOException
             {
                 obligationFulfiller.fulfill( txId );
             }
+            catch ( IllegalStateException e )
+            {
+                throw new ComException( "Failed to pull updates", e );
+            }
             catch ( InterruptedException e )
             {
                 throw new IOException( e );
```

Comment:
I think you're ignoring the lifecycle here. Before we shut down Jetty we shut down Neo4j database, which will kill all existing transactions and ensure no new ones are created. So I don't think that interrupt by Jetty does anything meaningful to anything inside of Neo4j.